app/experimenter/nimbus-ui/.storybook/preview.js

```diff
@@ -3,3 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import '../src/styles/index.scss';
+
+export const parameters = {
+  layout: "fullscreen"
+};
```

Comment:
Doesn't look like this file or the SVG has a newline, might need to tweak a lint config.

Reply:
I know @jaredlockhart like's to obliterate newlines, maybe it's worth a team consensus 😈